src/components/Trade/TradeConfirm/ReceiveSummary.tsx

```diff
@@ -26,17 +27,21 @@ type ReceiveSummaryProps = {
   intermediaryTransactionOutputs?: AmountDisplayMeta[]
   fiatAmount?: string
   amountBeforeFeesCryptoPrecision?: string
-  protocolFees?: AmountDisplayMeta[]
+  protocolFees?: Record<AssetId, ProtocolFee>
   shapeShiftFee?: string
   slippage: string
   swapperName: string
 } & RowProps
 
-const parseAmountDisplayMeta = ({ amountCryptoBaseUnit, asset }: AmountDisplayMeta) => ({
-  symbol: asset.symbol,
-  chainName: getChainAdapterManager().get(asset.chainId)?.getDisplayName(),
-  amountCryptoPrecision: fromBaseUnit(amountCryptoBaseUnit, asset.precision),
-})
+const parseAmountDisplayMeta = (items: AmountDisplayMeta[]) => {
```

Comment:
Not introduced by this PR but any rationale why this lives outside of component scope and not inside a `useCallback` within `ReceiveSummary`, or can we safely move it there?

Reply:
no reason, fixed :)